screens/Registration/InterestsScreen/styles.ts

```diff
@@ -57,36 +57,35 @@ const styles = StyleSheet.create({
   },
   footerButtonText: {
     color: black,
-    textAlign: 'center',
+    textAlign: "center",
   },
 
-
   image: {
-    alignSelf: 'center',
+    alignSelf: "center",
     height: 75,
```

Comment:
I think you should use percentages for sizes here.

Reply:
Okay I'll do it!